flink-end-to-end-tests/flink-confluent-schema-registry/pom.xml

```diff
@@ -53,6 +53,11 @@ under the License.
 			<artifactId>flink-connector-kafka</artifactId>
 			<version>3.0.0-1.17</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-shaded-guava</artifactId>
+			<version>30.1.1-jre-16.1</version>
```

Comment:
shouldn't it be 31.xx ?

Reply:
yes, i see...
then would it make sense to have a comment that this version should be in sync with kafka-connector's?

Reply:
I'll add a comment when merging :)